WordPress/src/main/java/org/wordpress/android/userflags/resolver/UserFlagsResolver.kt

```diff
@@ -34,35 +28,21 @@ class UserFlagsResolver @Inject constructor(
         }
         userFlagsAnalyticsTracker.trackStart()
         appPrefsWrapper.saveIsFirstTryUserFlagsJetpack(false)
-        val userFlagsResultCursor = getUserFlagsResultCursor()
-        if (userFlagsResultCursor != null) {
-            val userFlags = queryResult.getValue<Map<String, Any?>>(userFlagsResultCursor) ?: emptyMap()
-            if (userFlags.isNotEmpty()) {
-                val success = updateUserFlags(userFlags)
-                if (success) {
-                    userFlagsAnalyticsTracker.trackSuccess()
-                    onSuccess()
-                } else {
-                    userFlagsAnalyticsTracker.trackFailed(ErrorType.UpdateUserFlagsError)
-                    onFailure()
-                }
+        val (userFlags) = localMigrationContentResolver.getDataForEntityType<UserFlagsData>(UserFlags)
+        if (userFlags.isNotEmpty()) {
+            val success = updateUserFlags(userFlags)
+            if (success) {
+                userFlagsAnalyticsTracker.trackSuccess()
+                onSuccess()
             } else {
-                userFlagsAnalyticsTracker.trackFailed(ErrorType.NoUserFlagsFoundError)
+                userFlagsAnalyticsTracker.trackFailed(ErrorType.UpdateUserFlagsError)
                 onFailure()
             }
         } else {
-            userFlagsAnalyticsTracker.trackFailed(ErrorType.QueryUserFlagsError)
-            onFailure()
-        }
-    }
-
-    private fun getUserFlagsResultCursor(): Cursor? {
-        val wordpressUserFlagsUriValue =
-                "content://${wordPressPublicData.currentPackageId()}.${UserFlagsProvider::class.simpleName}"
-        return contentResolverWrapper.queryUri(
-                contextProvider.getContext().contentResolver,
-                wordpressUserFlagsUriValue
-        )
+                userFlagsAnalyticsTracker.trackFailed(ErrorType.NoUserFlagsFoundError)
+                onFailure()
+            }
+//            userFlagsAnalyticsTracker.trackFailed(ErrorType.QueryUserFlagsError)
```

Comment:
Should we uncomment or remove this line before merging?

Reply:
Handled w/ https://github.com/wordpress-mobile/WordPress-Android/pull/17423/commits/85553bb376299203fdab1ffd07a9aa40a945203c